packages/components/src/internal/components/notifications/ServerActivityList.tsx

```diff
@@ -48,8 +48,8 @@ export class ServerActivityList extends React.PureComponent<Props> {
 
             const detailsDisplay = isError ? resolveErrorMessage(details) : details;
             return (<>
-                <span className={'server-notifications-item-subject'}>{subject}</span>
-                {detailsDisplay && <span className={'server-notifications-item-details'}>{detailsDisplay}</span>}
+                <span className={'server-notifications-item-subject'} dangerouslySetInnerHTML={{ __html: subject }} />
+                {detailsDisplay && <span className={'server-notifications-item-details'} dangerouslySetInnerHTML={{ __html: detailsDisplay }} />}
```

Comment:
I assume the subject and detailsDisplay values here can't be changed so that the server doesn't HTML encode them before sending?

Reply:
that's good thought, they can be changed on server side for some notifications (those that are true text/plain, but not those text/html). I imagine that notifications shown for LKSM will all be generated in LKSM, so we can control those to be plain text only. I'll make the change to switch msg generation to be text/plain, and also update the client side. We should probably avoid the use of dangerouslySetInnerHTML whenever possible